cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/MetricsTable.java

```diff
@@ -84,6 +86,18 @@
   void delete(byte[] row, byte[][] columns);
 
   /**
+   * checks the expected value before performing a put
```

Comment:
if you have @param, then it should actually describe what the parameter means. Also, start a Javadoc with an upper case letter, and end the sentence with a period?